eslint-plugin/src/index.ts

```diff
@@ -16,7 +16,7 @@ interface IPlugin {
 
 export const plugin: IPlugin = {
   rules: {
-    "tsdoc-comments": {
+    "syntax": {
```

Comment:
Is the rule name "syntax?" Or does it get the name from the package name?

Reply:
ESLint has a weird way of calculating rule names by deleting text from the NPM package name. They really really do not trust plugin authors to avoid naming conflicts between rules.  Anyway the name is "tsdoc/syntax".

Reply:
Can you leave a comment about that? Because that's really weird.